packages/graphql/src/translate/queryAST/ast/operations/interfaces/InterfaceReadOperation.ts

```diff
@@ -103,4 +107,8 @@ export class InterfaceReadOperation extends Operation {
     public addSort(...sortElement: Sort[]): void {
         this.sortFields.push(...sortElement);
     }
+
+    protected getSortFields(context: QueryASTContext, target: Cypher.Variable): SortField[] {
+        return this.sortFields.flatMap((sf) => sf.getSortFields(context, target, false));
+    }
```

Comment:
I think the name of this function this may lead to a problem in the future

`getSortFields` is the name used in the contract for sort, which is "public" in class `Sort`

In this case, we have a protected function with the same name. At the moment is not a problem, but in the future we will probably try to merge all the interfaces to have a common contract between all nodes

I think we should consider renaming this protected function, to avoid confusion/conflicts with the contract in the future

Reply:
Sure, I agree! Just a thing, this name and behavior mimic what we have in the class InterfaceConnectionReadOperation, I tried to keep everything as close as possible between these classes so that will be easier to unify later!

Reply:
Oh, that's great, is just so we can distinguish that this is not yet the same interface :)